pom.xml

```diff
@@ -27,10 +27,17 @@
         <!-- Dependency versions -->
         <groovy.version>2.4.17</groovy.version>
         <junit.version>4.12</junit.version>
-        <jenkins-pipeline-unit.version>1.1</jenkins-pipeline-unit.version>
+        <jenkins-pipeline-unit.version>1.3</jenkins-pipeline-unit.version>
         <groovy-eclipse-compiler.version>3.4.0-01</groovy-eclipse-compiler.version>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
```

Comment:
It does not exist https://repo.maven.apache.org/maven2/com/lesfurets/jenkins-pipeline-unit/1.3 yet

Reply:
Still not there

Reply:
As long as the release is not published to the repo.maven then we need to use `repo.jenkins-ci.org`